src/iomusxml.cpp

```diff
@@ -2165,4 +2228,20 @@ std::string MusicXmlInput::ConvertKindToSymbol(std::string value)
         return "";
 }
 
+bool MusicXmlInput::NotInEndingStack(std::string measureN_)
```

Comment:
Please use `const std::string &measureN` as parameter (and do not make a copy in the method if not necessary)

Reply:
Ok.